mutis/tests/test_signal.py

```diff
@@ -1,16 +1,61 @@
 import pytest
 import numpy as np
+from numpy.testing import assert_allclose
 from mutis.signal import Signal
 
 
 @pytest.fixture
-def profile():
-    p = {"times": np.linspace(10, 20, 100)}
-    p["values"] = 0.5 * np.sin(1 * p["times"]) + 0.5 * np.sin(6*p["times"]) + 1
-    return p
+def signal():
+    times = np.linspace(10, 20, 100)
+    values = 0.5 * np.sin(1 * np.linspace(10, 20, 100)) + 0.5 * np.sin(6 * np.linspace(10, 20, 100)) + 1
 
+    return {
+        "fail": Signal(times, values, "test"),
+        "samp": Signal(times, values, "lc_gen_samp"),
+        "psd_nft": Signal(times, values, "lc_gen_psd_nft"),
+        "psd_fft": Signal(times, values, "lc_gen_psd_fft"),
+        "psd_lombscargle": Signal(times, values, "lc_gen_psd_lombscargle"),
+        "ou": Signal(times, values, "lc_gen_ou"),
+    }
 
-def test_generation(profile):
-    signal = Signal(profile["times"], profile["values"], "lc_gen_samp")
+
+@pytest.fixture
+def ou_params():
+    return {
+        "theta": 100,
+        "mu": 1,
+        "sigma": 10
+    }
+
+
+def test_gen_synth(signal, ou_params):
```

Comment:
same question as above (maybe I'm not familiar with this kind of tests...?)

Reply:
I have added some asserts to check at least the correct shape of the returned array.